icons/SOURCES.md

```diff
@@ -4,8 +4,9 @@
 |---|---|---|---|
 | hamburger-menu.svg | [wikimedia](https://commons.wikimedia.org/wiki/File:Hamburger_icon.svg) | [Timothy Miller](https://tmthymllr.com/) | [SPDX: CC-BY-SA-3.0](https://spdx.org/licenses/CC-BY-SA-3.0.html) |
 | lock.svg | [fxemoji](https://github.com/mozilla/fxemoji/blob/gh-pages/svgs/objects/u1F512-lock.svg) | Mozilla Foundation | [SPDX: CC-BY-4.0](https://spdx.org/licenses/CC-BY-4.0.html) |
-| turtle-silhouette.svg | [fxemoji](https://github.com/mozilla/fxemoji/blob/gh-pages/svgs/nature/u1F422-turtle.svg) (derived from) | Mozilla Foundation | [SPDX: CC-BY-4.0](https://spdx.org/licenses/CC-BY-4.0.html) |
+| ratio.svg | [forum.transmissionbt](https://forum.transmissionbt.com/viewtopic.php?p=42307#p42307) | [Jakub Steiner](https://jimmac.eu/) | [SPDX: CC-BY-SA-2.5](https://spdx.org/licenses/CC-BY-SA-2.5.html) |
```

Comment:
There's no ratio.svg, and yin-yang.svg is already listed in this table down below, so please remove this. There are however other files with no attribution but that shouldn't be your concern.

Reply:
Done. Thanks!